pyproject.toml

```diff
@@ -11,6 +11,8 @@ dependencies = [
     "ai-exchange>=0.8.0",
```

Comment:
accidentally committed this

Reply:
Shall we change the version to `0.9.0`? (depends on the new release of your `exchange` PR)